pages/index.js

```diff
@@ -24,35 +26,44 @@ let ingredients = [
 ];
 
 export default function Home() {
+  const [ingredients, setIngredients] = useState(defaultIngredients);
+
   function editBagle(ingredient) {
-    if (ingredient.chosen) {
-      ingredient.chosen = false;
-    } else {
-      ingredient.chosen = true;
-    }
+    ingredient.chosen = !ingredient.chosen;
   }
 
   return (
     <div>
+      <Head>
+        <title>Parkbench Bagels</title>
+      </Head>
+
       <h1>Parkbench Bagels</h1>
 
       <h2>Build your own Bagle!</h2>
+      <form
+        onSubmit={(e) => {
+          // only for Testing
+          e.preventDefault();
+          console.log(e.t);
+        }}
+      >
+        <ul>
+          {ingredients.map((ingredient) => {
+            return (
+              <li key={ingredient.name}>
+                {ingredient.name}
+                <input
+                  type="checkbox"
+                  onChange={() => editBagle(ingredient)}
+                ></input>
+              </li>
+            );
+          })}
+        </ul>
 
-      <ul>
-        {ingredients.map((ingredient) => {
-          return (
-            <li key={ingredient.name}>
-              {ingredient.name}
-              <input
-                type="checkbox"
-                onChange={() => editBagle(ingredient)}
-              ></input>
-            </li>
-          );
-        })}
-      </ul>
-
-      <Button onClick={() => {}}>Submit</Button>
+        <Button>Submit</Button>
```

Comment:
I would recommend to always add the type of your button to have code that is easier to understand and to reduce possible bugs as the default type of a button is always `submit`. Even if the default type is submit I would add this, but that is up to you to decide.